src/app/profile/my-spaces/my-spaces.component.ts

```diff
@@ -146,18 +146,16 @@ export class MySpacesComponent implements OnDestroy, OnInit {
   matchesFilter(space: Space, filter: Filter): boolean {
     let match = true;
     if (filter.field.id === 'name') {
-      match = space.attributes.name.match(filter.value) !== null;
+      match = space.attributes.name.toLocaleLowerCase().includes(filter.value.toLocaleLowerCase());
```

Comment:
I believe using "includes" Vs "match" breaks the patternfly pattern.

Reply:
Ah sure, I can revert the changes in that case.